README.md

```diff
@@ -1,30 +1,168 @@
-# website
-Kubeflow's public website
+# Creating and updating the Kubeflow docs
 
-## Development
+Welcome to the GitHub repository for Kubeflow's public website. The docs are
+hosted at https://www.kubeflow.org.
 
-### Getting started
-macOS...sorry :-)
+We use [Hugo](https://gohugo.io/) to format and generate our website. Hugo is
+an open-source static site generator that provides us with templates, content
+organisation in a standard directory structure, and a website generation engine.
+You write the pages in Markdown, and Hugo wraps them up into a website.
 
-`brew install hugo`
+## Install Hugo
 
-`hugo serve -D`
+See the [Hugo installation guide][hugo-install]. Here are some examples:
 
-### New content
-`hugo new blog/another-blog-post.md`
+### Mac OS X:
 
-`hugo new doc/add-this-doc.md`
+```
+brew install hugo
```

Comment:
I see. I didn't know we automatically generate a preview. How/where can I see that preview? I guess that may become clear the first time I submit a PR for a web page, but as someone putting together my first website PR I had no idea that will happen. I therefore assumed I had to run my own Hugo server. I've spent quite some time reading the Hugo docs, and then figuring out how to fit them into our own situation. It'd be good to help others avoid doing the same work, by adding this info to the guide.

Reply:
I'll figure out how the generated preview happens for a PR, and include it in this update.
/hold

Reply:
OK, I've figured out I should probably add "draft = true" to the page front matter, to indicate to Hugo that this is a draft. Done that. But even after reading all the Hugo docs and our config files, I can't figure out what the URL is where the draft matter should show up. Please would you ping me the URL? In case it's useful, here's my PR with draft content: https://github.com/kubeflow/website/pull/151